src/HttpHeader.h

```diff
@@ -59,6 +59,8 @@ class HttpHeaderEntry
     void packInto(Packable *p) const;
     int getInt() const;
     int64_t getInt64() const;
+    /// packed length, including ": " and crlf
```

Comment:
I know you are copying this, but let's polish a little:

```suggestion
    /// packed length, including ": " and CRLF
```